src/cli/javascript/codec.rs

```diff
@@ -322,7 +322,7 @@ mod test {
     #[test]
     fn test_command_from_invalid_object() {
         let runtime = Runtime::new().unwrap();
-        let context = Context::base(&runtime).unwrap();
+        let context = Context::full(&runtime).unwrap();
```

Comment:
tests are failing without this change, see https://github.com/DelSkayn/rquickjs/issues/392